src/components/DonutChart/DonutChart.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled from 'styled-components'
+import { VictoryPie } from 'victory'
+import { mapTripToCarbonActivityTypeToDonutChartLabel } from '../utils/mappings/mapTripToCarbonActivityTypeToDonutChartLabel'
+
+export default function DonutChart({ footprintData }) {
+  const colors = [
```

Comment:
Dusk und woodland sehen bei mir fast gleich aus, vielleicht findest du noch andere Farben, die sich etwas mehr unterscheiden! Als die Sonne eben weg war hab ich einen Unterscheid gesehen, aber als ich anfing den Kommentar hier zu schreiben waren die beiden Farben kaum voneinander zu trennen 😎

Reply:
Guter Hinweis, dann schau ich mal, welche Farbe anstatt Dusk noch in meine Fabrpalette passen würde :)